src/mavsdk/core/tcp_connection.cpp

```diff
@@ -80,7 +81,16 @@ ConnectionResult TcpConnection::setup_port()
     struct sockaddr_in remote_addr {};
     remote_addr.sin_family = AF_INET;
     remote_addr.sin_port = htons(_remote_port_number);
-    remote_addr.sin_addr.s_addr = inet_addr(_remote_ip.c_str());
+
+    struct hostent* hp;
+    hp = gethostbyname(_remote_ip.c_str());
```

Comment:
```suggestion
    struct hostent* hp; = gethostbyname(_remote_ip.c_str());
```

As I read it, this doesn't need cleaning up, right?